src/database/firebase.tsx

```diff
@@ -18,6 +19,115 @@ const config = {
   appId: APP_ID,
 };
 
-firebase.initializeApp(config);
+!firebase.apps.length ? firebase.initializeApp(config) : firebase.app();
 
+const database = firebase.firestore();
+const treeCollection = database.collection('trees');
+const commentCollection = database.collection('comments');
+const additionalCollection = database.collection('additional');
+
+/*
+ * checkID validates that this ID exists in the `trees` table.
+ */
+export const checkID = async function (id: string): Promise<boolean> {
+  await treeCollection
+    .doc(id)
+    .get()
+    .then(doc => {
+      return doc.exists;
+    })
+    .catch(e => {
+      console.log('Error getting document:', e);
```

Comment:
use console.warn(..) here for linting and also so its classified as an error / warning in the console